pnet_datalink/Cargo.toml

```diff
@@ -14,6 +14,7 @@ libc = "0.2"
 ipnetwork = "0.12"
 pnet_base = { path = "../pnet_base" }
 pnet_sys = { path = "../pnet_sys" }
+pcap = { version = "0.6", optional = true }
 
 pcap = { version = "0.6", optional = true }
```

Comment:
Now we ended up with two `pcap` here. It was also added here by the PR merged by @mrmonday recently. He specifically commented that he had to change those things for his PR, but also did not want to step on your toes. So he did not mean to copy your stuff, just get his stuff working :)